src/main/java/net/sf/jabref/gui/desktop/JabRefDesktop.java

```diff
@@ -432,4 +434,38 @@ public static void openBrowser(String url) throws IOException {
         ExternalFileType fileType = ExternalFileTypes.getInstance().getExternalFileTypeByExt("html");
         openExternalFilePlatformIndependent(fileType, url);
     }
+
+    public static void openConsole(JabRefFrame frame) throws IOException {
```

Comment:
Why should the JabRefDesktop class has a reference to the whole frame? Would a File reference not suffice of the folder in which the console should be openend?